server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SendMDNMethodTest.java

```diff
@@ -93,11 +99,37 @@ public void setup() throws Throwable {
         dataProbe.addUser(BOB, BOB_PASSWORD);
         mailboxProbe.createMailbox("#private", USERNAME, DefaultMailboxes.INBOX);
         accessToken = HttpJmapAuthentication.authenticateJamesUser(baseUri(), USERNAME, PASSWORD);
+        bobAccessToken = HttpJmapAuthentication.authenticateJamesUser(baseUri(), BOB, BOB_PASSWORD);
+        await();
+
+        String messageCreationId = "creationId";
```

Comment:
Can you extract this provisioning to a method & call it on each test methods if needed.

Reply:
this method can return the created MessageId.